lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -57,6 +57,21 @@ def stub_email_alert_api_has_subscription(id, frequency, optional_overrides = {}
           .to_return(status: 200, body: response)
       end
 
+      # Stubs the API responses as if each subscription happened in the order they are passed
+      def stub_email_alert_api_has_subscriptions(*subscriptions)
+        subscriptions.map! do |id, frequency, optional_overrides|
```

Comment:
Rather that specifying the parameters in the block definition, you can just use 
`subscriptions.map! do |subscription|` and use `subscription[:id]`  to populate the id.

Reply:
I don't think you can? `subscription` is a tuple of two strings and a hash:

```ruby
[
  'abc123',
  'daily',
  {
    ended: false,
    subscription_reason: 'etc',
  }
]
```

Reply:
How would you get the `id` for `subscriptions[0]`? That's what you'd use in the loop.

Reply:
If I do `subscriptions.map! do |subscription|` then inside the loop it will either be a):

```ruby
id, frequency, optional_overrides = subscription
```

or b):

```ruby
id = subscription[0]
frequency = subscription[1]
optional_overrides = subscription[2]
```

Which would be best?

Reply:
Could `subscriptions` be changed to be an array of hashes?
e.g.
```
[
  {
    id: 'abc123', 
    frequency: 'daily',
    ended: false,
    subscription_reason: 'etc',
  }
]
```
It might make it easier to call from a test.